### PR TITLE
CL mobile fixes

### PR DESCRIPTION
**Pull request by jonator**
### ClickUp Task

[ClickUp Task URL](PASTE_CLICKUP_TASK_URL_HERE)

### CODE REVIEW

**Review by arispech — APPROVED** [review]
LGTM

**Comment by jonator** [comment]
QA approves

**Review by ValarDragon — DISMISSED** [review]

File: packages/web/components/pool-detail/base-pool.tsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ export const BasePoolDetails: FunctionComponent<{
                     {t("pool.swapFee")}
                   </span>
                   <h4 className="text-osmoverse-100">
-                    {new RatePretty(pool.swapFee).maxDecimals(0).toString()}
+                    {new RatePretty(pool.swapFee).maxDecimals(2).toString()}
```
Comment (ValarDragon) [inline packages/web/components/pool-detail/base-pool.tsx:90]: Why did this change?

**Review by ValarDragon — APPROVED** [review]
CSS changes LGTM. Confused by the swap fee display change

File: packages/web/components/pool-detail/base-pool.tsx (hunk 1 above)
Comment (ValarDragon) [reply packages/web/components/pool-detail/base-pool.tsx:90]: Ah this would never work, since its always 1 to 2 decimal points.

Comment (ValarDragon) [reply packages/web/components/pool-detail/base-pool.tsx:90]: Is there somewhere that makes this a mobile only component, or did this not work on desktop either?

Comment (jonator) [reply packages/web/components/pool-detail/base-pool.tsx:90]: We have `isMobile` that we could use

**Pull request merged**